packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -1,14 +1,22 @@
 :local {
   .demographics-banner {
     background: var(--terra-demographics-banner-background, #004c76);
```

Comment:
hmm, we can probably change this to `background-color`

Reply:
Isn't there a need for a gradient on this for clinical though?

Reply:
we have `background-image` defined below

Reply:
https://github.com/cerner/terra-core/pull/2133/commits/a72fdf778b090b7cf5ac106e1f1ecd0bd1456945#diff-2267f9efeef50e2d1511d46f3a178e10R3